### DISCUSSION TITLE
Is there a way to remove year in header title

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [gpbl/react-day-picker#1842] [Support] [answered]
I want to just set the title to "July"

![image](https://github.com/gpbl/react-day-picker/assets/22732118/e1032d51-3e4a-4cb0-b66b-8fc39e369914)

**Comment by gpbl** [comment, marked as answer]
@isaacdarcilla yes, use a `formatCaption` formatter:

https://codesandbox.io/s/pensive-poitras-5dn8yg?file=/App.tsx

```tsx
import { DateFormatter, DayPicker } from "react-day-picker";

import { format } from "date-fns";

const formatCaption: DateFormatter = (month, options) => {
  return format(month, "LLLL", options);
};

function App() {
  return <DayPicker formatters={{ formatCaption }} />;
}
```

(Consider hiding the year is not a good practice for birthday dates: https://www.smashingmagazine.com/2021/05/frustrating-design-patterns-birthday-picker/)
